examples/xfixes-selection-notify.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python3
+#
+# examples/xfixes-selection-notify.py -- demonstrate the XFIXES extension
+# SelectionNotify event.
+#
+#    Copyright (C) 2019
+#      Tony Crisci <tony@dubstepdish.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation; either version 2.1
+# of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the
+#    Free Software Foundation, Inc.,
+#    59 Temple Place,
+#    Suite 330,
+#    Boston, MA 02111-1307 USA
+
+# Python 2/3 compatibility.
+from __future__ import print_function
+
+import sys
+import os
+import time
+
+# Change path so we find Xlib
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+
+from Xlib.display import Display
+from Xlib.ext import xfixes
+
+def main(argv):
+    if len(sys.argv) < 2 or len(sys.argv) > 3:
```

Comment:
`if len(sys.argv) != 2:` is more precise and shorter. Now it includes `== 3` as well but `sys.argv[2]` is not used below.

Reply:
:+1: